ionic/__main__.py

```diff
@@ -1,15 +1,18 @@
 import argparse
 import asyncio
 import datetime as dt
+from datetime import datetime, timedelta 
 import random
 import sys
 from typing import List, Union
+from decimal import Decimal, getcontext
 
 import dateparser
 import hikari as h
 import lightbulb as lb
 import regex as re
 import sqlalchemy as sql
+from sqlalchemy.sql.traversals import ColIdentityComparatorStrategy
```

Comment:
This doesn't seem to be used @cchollar ,
Is there any reason this is here?

Reply:
I have no idea why this is here, my pyright install must have tried to put it in lol I will get this removed

Reply:
No worries!